### ISSUE TITLE
Error when running through WebdriverIO testrunner

### ISSUE THREAD

**Issue by klamping** [applitools/eyes.webdriverio.javascript#10]
I get an error when trying to run my tests in [the WebdriverIO Testrunner](http://webdriver.io/guide/testrunner/gettingstarted.html) mode.

Here's a quick repo I put together with the official example, plus my adaption to run it in the testrunner:
https://github.com/klamping/test-app-wdio/tree/master

The screenshots upload fine, but I get the following odd error:
```
ServerConnector.stopSession - unexpected status (OK)
Error: ServerConnector.stopSession - unexpected status (OK)
    at sendLongRequest.then.response (/Users/klamping/Sites/test-app-wdio/node_modules/@applitools/eyes.sdk.core/src/server/ServerConnector.js:271:19)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
```

**Comment by danielputerman** [comment]
Thanks! Checking it out.

**Comment by danielputerman** [comment]
fixed in #12

**Comment by danielputerman** [comment]
Will be available in the next version update.

**Comment by danielputerman** [comment]
@klamping fix is available in version 1.1.0 in npm :)

**Comment by klamping** [comment]
Just checked it out and it works great! Can't wait to integrate this in to more things!
